README.md

```diff
@@ -41,45 +41,45 @@ For documentation for Cadl language, see https://microsoft.github.io/cadl.
 
    It is recommended to have npm 7+. To update npm run `npm install -g npm`
 
-2. Install Cadl compiler and libraries:
+2. Install TypeSpec compiler and libraries:
 
 ```bash
    npm init -y
-   npm install -g @cadl-lang/compiler
+   npm install -g @typespec/compiler
 ```
 
-If you do not wish to install the compiler globally with `-g` flag, you will need to install it locally once in every Cadl project folder. You would also need to prefix every Cadl run command with `npx`. See [npx documentation](https://docs.npmjs.com/cli/v7/commands/npx)
+If you do not wish to install the compiler globally with `-g` flag, you will need to install it locally once in every TypeSpec project folder. You would also need to prefix every TypeSpec run command with `npx`. See [npx documentation](https://docs.npmjs.com/cli/v7/commands/npx)
 
 ```bash
-    npx cadl init
-    npx cadl compile
+    npx tsp init
+    npx tsp compile
 ```
 
-3. Install the Cadl extension for your editor of choice:
+3. Install the TypeSpec extension for your editor of choice:
 
    - [Instructions for Visual Studio](#installing-visual-studio-extension)
    - [Instructions for Visual Studio Code](#installing-vs-code-extension)
 
-### Creating Cadl project
+### Creating TypeSpec project
 
-1. Create a folder for your new Cadl project
+1. Create a folder for your new TypeSpec project
 
-2. Initialize a Cadl project.
+2. Initialize a TypeSpec project.
 
-   - Run `cadl init` > Select `Generic Rest API` template with `@cadl-lang/rest` and `@cadl-lang/openapi3` libraries checked.
-   - Run `cadl install` to install node package dependencies.
+   - Run `tsp init` > Select `Generic Rest API` template with `@typespec/rest` and `@typespec/openapi3` libraries checked.
+   - Run `typespec install` to install node package dependencies.
 
-3. Open the folder in your editor and edit `main.cadl`
+3. Open the folder in your editor and edit `main.tsp`
 
-4. Follow our [documentation](https://microsoft.github.io/cadl) to get started writing Cadl!
+4. Follow our [documentation](https://microsoft.github.io/typespec) to get started writing TypeSpec!
 
-5. Once you're ready to compile your Cadl to Swagger, save the file and type this at the command prompt in your project folder:
+5. Once you're ready to compile your TypeSpec to Swagger, save the file and type this at the command prompt in your project folder:
 
    ```bash
-   cadl compile .
+   typespec compile .
    ```
 
-   This will compile the Cadl files in the project folder into one output file: `.\cadl-output\openapi.json`.
+   This will compile the TypeSpec files in the project folder into one output file: `.\typespec-output\openapi.json`.
```

Comment:
Another minor one, but wonder if we want `tsp-output`, something shorter, as default.